### PR TITLE
fix-bff-configs: Draft missing BFF configs

### PR DESCRIPTION
**Pull request by galales**

### CODE REVIEW

File: kubernetes/configs/dev/backend_for_frontend.sh [hunk 1]
```diff
@@ -18,3 +18,6 @@ BFF_RATE_LIMITER_MAX_REQUESTS="10"
 BFF_RATE_LIMITER_BURST_PERCENTAGE="1.0"
 BFF_RATE_LIMITER_RATE_INTERVAL="1.second"
 BFF_RATE_LIMITER_TIMEOUT="300.milliseconds"
+
+SUPPORT_SAML_CALLBACK_URL=""
+SUPPORT_SAML_CALLBACK_ERROR_URL=""
```
Comment (galales) [inline kubernetes/configs/dev/backend_for_frontend.sh]: @beetlecrunch can you provide these urls?

File: kubernetes/configs/dev/commons.sh [hunk 2]
```diff
@@ -42,5 +42,6 @@ PRIVACY_NOTICES_DYNAMO_TABLE_NAME="interop-privacy-notices-dev"
 PRIVACY_NOTICES_ACCEPTANCE_DYNAMO_TABLE_NAME="interop-privacy-notices-acceptances-dev"
 PRIVACY_NOTICES_UPDATER_PRIVACY_POLICY_UUID="0df21ff6-3e8f-4320-af8f-23dea9135d57"
 PRIVACY_NOTICES_UPDATER_TERMS_OF_SERVICE_UUID="6bf8412a-41a7-41a0-82dc-26286ce61b1a"
+PRIVACY_NOTICES_CONTAINER=""
```
Comment (galales) [inline kubernetes/configs/dev/commons.sh]: @beetlecrunch do we have this too?

File: kubernetes/configs/dev/backend_for_frontend.sh (hunk 1 above)
Comment (beetlecrunch) [reply kubernetes/configs/dev/backend_for_frontend.sh]:
```suggestion
SUPPORT_SAML_CALLBACK_URL="https://selfcare.dev.interop.pagopa.it/ui/it/assistenza/scelta-ente#id="
SUPPORT_SAML_CALLBACK_ERROR_URL="https://selfcare.dev.interop.pagopa.it/ui/it/assistenza/errore"
```

File: kubernetes/configs/dev/commons.sh (hunk 2 above)
Comment (galales) [reply kubernetes/configs/dev/commons.sh]: copied from [here](https://github.com/pagopa/pdnd-interop-platform-deployment/pull/296/files#diff-e74e70e12a4efa4ed611fd20d4a8734c8b4f276447c4d53f04e155055c3b334bR5)

**Review by beetlecrunch — APPROVED** [review]

**Review by micdes-pagopa — APPROVED** [review]

**Pull request merged**